pages/common/eslint.md

```diff
@@ -0,0 +1,19 @@
+# eslint
+
+> Find problematic patterns or code that doesn’t adhere to certain style guidelines.
```

Comment:
I think the main heading in the site's page is a better description for the tool - " A pluggable linting utility for JavaScript and JSX"